app/components/shared/form/Datefield.js

```diff
@@ -29,6 +24,12 @@ class Datefield extends React.Component {
     }
   }
 
+  state = {
```

Comment:
https://facebook.github.io/react/docs/react-component.html#constructor.
> The constructor for a React component is called before it is mounted.
If you want to have state ready for when the component is mounted, you would set them within the `constructor` method.

Reply:
this is the rule: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md

I don't see any example in the Facebook docs that have a separate state() function as a class member so I feel like it should be deleted

Reply:
This is an example of a component changing with no tests that is not used in app folder